manual/src/macros.hva

```diff
@@ -180,6 +181,8 @@
 \newcommand{\ocamlcomment}{\@span{class="ocamlcomment"}}
 \newcommand{\ocamlstring}{\@span{class="ocamlstring"}}
 
+\newcommand{\?}{\@span{class=ocamlprompt}\#}
+\newstyle{.ocamlprompt}{color:black;}
```

Comment:
The previous definition of `\? `also used `\normalsize`, is anything lost here?

Reply:
As seen in my table above, this was translated in css to
```css
color:black;
```
thus nothing has been lost as far as I can see.